sherpa/astro/xspec/src/_xspec.cc

```diff
@@ -404,9 +408,8 @@ int _sherpa_init_xspec_library()
     FPATDV(atomdbVersion1210);
 #endif
 
-  } catch(...) {
+  } catch(std::exception& e) {
```

Comment:
Oops - I didn't mean to commit this change.

Reply:
Although I think it might make sense to be able to provide the actual error message to the user, since it may not be due to the HEADAS env. variable setting.